src/app/backend/dashboard.go

```diff
@@ -32,6 +32,7 @@ var (
 )
 
 func main() {
+	//TODO(maciaszczykm): Resolve issue with logs (ERROR: logging before flag.Parse).
```

Comment:
I agree. I've used `glog`, because it was already used in `dashboard.go` and there are logging levels available. Let's use golang log as you said.